src/index.html

```diff
@@ -11,12 +11,116 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./normalize.css"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="images/logo.png"
+        class="moyo_logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo_logo"
+          class="moyo_img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
Let's fix the last thing

Following `BEM` we need to write class something like this:
- `block__element--modification`
- so it need to `nav__list` with double `_`

Just read this [article](https://en.bem.info/methodology/naming-convention/#naming-rules) about naming